docs/data/routes/docs/nextjs/getting-started/options/en.md

```diff
@@ -3,4 +3,22 @@ name: options
 routeTemplate: ./data/component-templates/article.yml
 title: Getting Started options overview
 ---
-# Getting Started options overview
\ No newline at end of file
+#  Development environment options for JSS Next.js projects
+
+Sitecore offers two paths for setting up your development environment, based on your role and operating system. 
+
+## Full-stack development
```

Comment:
Should we state that this is optimized for a Sitecore-first workflow as well or is that getting too deep/confusing here?